presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java

```diff
@@ -234,6 +238,7 @@ public OrcWriter(
                 .build();
         this.metadataWriter = new CompressedMetadataWriter(orcEncoding.createMetadataWriter(), columnWriterOptions, Optional.empty());
         this.hiveStorageTimeZone = requireNonNull(hiveStorageTimeZone, "hiveStorageTimeZone is null");
+        this.enableExecutionBasedMemoryAccounting = enableExecutionBasedMemoryAccounting;
```

Comment:
From OrcWriterOptions this should be transferred to the ColumnWriterOptions. If it is funneled inside the ColumnWriterOptions, it would be available to all the writers.